pkgs/desktops/gnome-3/apps/gnome-documents/default.nix

```diff
@@ -31,6 +31,11 @@ stdenv.mkDerivation rec {
     gnome-desktop libzapojit libgepub
   ];
 
+  patches = [
+    # fix RPATH to libgd
+    ./libgd.patch
```

Comment:
Could you fetch https://gitlab.gnome.org/GNOME/gnome-documents/commit/d18a92e0a940073ac766f609937539e4fc6cdbb7.patch?

Reply:
Can do :+1: